domains/naf.is-cool.dev.json

```diff
@@ -10,7 +10,7 @@
     },
 
     "record": {
-        "A": ["5.223.47.82"]
+        "A": ["103.161.181.131"]
```

Comment:
Unable to access site. Make sure you're site is accessible or previewable.

Reply:
Sorry, I forgot to mention that it’s the IP of my minecraft server.

IP: **103.161.181.131**:25141